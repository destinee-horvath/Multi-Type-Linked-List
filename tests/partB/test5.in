NEW 1
1
NEW 1
2
NEW 1
3
NEW 1
4
NEW 1
5
NEW 1
6
NEW 1
7
NEW 1
8
NEW 1
9
NEW 1
10
NEW 1
11
NEW 1
12
NEW 1
13
NEW 1
14
NEW 1
15
NEW 1
16
NEW 1
17
NEW 1
18
NEW 1
19
NEW 1
20
DELETE 0 0
DELETE 1 0
DELETE 2 0
DELETE 3 0
DELETE 4 0
DELETE 5 0
DELETE 6 0
DELETE 7 0
DELETE 8 0
DELETE 9 0
DELETE 10 0
DELETE 11 0
DELETE 12 0
DELETE 13 0
DELETE 14 0
DELETE 15 0
DELETE 16 0
DELETE 17 0
DELETE 18 0
DELETE 19 0
DELETE 0 0
DELETE 1 0
DELETE 2 0
DELETE 3 0
DELETE 4 0
DELETE 5 0
DELETE 6 0
DELETE 7 0
DELETE 8 0
DELETE 9 0
DELETE 10 0
DELETE 11 0
DELETE 12 0
DELETE 13 0
DELETE 14 0
DELETE 15 0
DELETE 16 0
DELETE 17 0
DELETE 18 0
DELETE 19 0
REMOVE 0
REMOVE 1
REMOVE 2
REMOVE 3
REMOVE 4
REMOVE 5
REMOVE 6
REMOVE 7
REMOVE 8
REMOVE 9
REMOVE 10
REMOVE 11
REMOVE 12
REMOVE 13
REMOVE 14
REMOVE 15
REMOVE 16
REMOVE 17
REMOVE 18
REMOVE 19
INSERT 0 0 tmp
INSERT 1 0 tmp
INSERT 2 0 tmp
INSERT 3 0 tmp
INSERT 4 0 tmp
INSERT 5 0 tmp
INSERT 6 0 tmp
INSERT 7 0 tmp
INSERT 8 0 tmp
INSERT 9 0 tmp
INSERT 10 0 tmp
INSERT 11 0 tmp
INSERT 12 0 tmp
INSERT 13 0 tmp
INSERT 14 0 tmp
INSERT 15 0 tmp
INSERT 16 0 tmp
INSERT 17 0 tmp
INSERT 18 0 tmp
INSERT 19 0 tmp
