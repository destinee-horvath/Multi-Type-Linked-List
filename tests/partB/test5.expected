List 0: 1
List 1: 2
List 2: 3
List 3: 4
List 4: 5
List 5: 6
List 6: 7
List 7: 8
List 8: 9
List 9: 10
List 10: 11
List 11: 12
List 12: 13
List 13: 14
List 14: 15
List 15: 16
List 16: 17
List 17: 18
List 18: 19
List 19: 20
List 0: 
List 1: 
List 2: 
List 3: 
List 4: 
List 5: 
List 6: 
List 7: 
List 8: 
List 9: 
List 10: 
List 11: 
List 12: 
List 13: 
List 14: 
List 15: 
List 16: 
List 17: 
List 18: 
List 19: 
INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
List 0 has been removed.

Number of lists: 19
List 1
List 2
List 3
List 4
List 5
List 6
List 7
List 8
List 9
List 10
List 11
List 12
List 13
List 14
List 15
List 16
List 17
List 18
List 19
List 1 has been removed.

Number of lists: 18
List 2
List 3
List 4
List 5
List 6
List 7
List 8
List 9
List 10
List 11
List 12
List 13
List 14
List 15
List 16
List 17
List 18
List 19
List 2 has been removed.

Number of lists: 17
List 3
List 4
List 5
List 6
List 7
List 8
List 9
List 10
List 11
List 12
List 13
List 14
List 15
List 16
List 17
List 18
List 19
List 3 has been removed.

Number of lists: 16
List 4
List 5
List 6
List 7
List 8
List 9
List 10
List 11
List 12
List 13
List 14
List 15
List 16
List 17
List 18
List 19
List 4 has been removed.

Number of lists: 15
List 5
List 6
List 7
List 8
List 9
List 10
List 11
List 12
List 13
List 14
List 15
List 16
List 17
List 18
List 19
List 5 has been removed.

Number of lists: 14
List 6
List 7
List 8
List 9
List 10
List 11
List 12
List 13
List 14
List 15
List 16
List 17
List 18
List 19
List 6 has been removed.

Number of lists: 13
List 7
List 8
List 9
List 10
List 11
List 12
List 13
List 14
List 15
List 16
List 17
List 18
List 19
List 7 has been removed.

Number of lists: 12
List 8
List 9
List 10
List 11
List 12
List 13
List 14
List 15
List 16
List 17
List 18
List 19
List 8 has been removed.

Number of lists: 11
List 9
List 10
List 11
List 12
List 13
List 14
List 15
List 16
List 17
List 18
List 19
List 9 has been removed.

Number of lists: 10
List 10
List 11
List 12
List 13
List 14
List 15
List 16
List 17
List 18
List 19
List 10 has been removed.

Number of lists: 9
List 11
List 12
List 13
List 14
List 15
List 16
List 17
List 18
List 19
List 11 has been removed.

Number of lists: 8
List 12
List 13
List 14
List 15
List 16
List 17
List 18
List 19
List 12 has been removed.

Number of lists: 7
List 13
List 14
List 15
List 16
List 17
List 18
List 19
List 13 has been removed.

Number of lists: 6
List 14
List 15
List 16
List 17
List 18
List 19
List 14 has been removed.

Number of lists: 5
List 15
List 16
List 17
List 18
List 19
List 15 has been removed.

Number of lists: 4
List 16
List 17
List 18
List 19
List 16 has been removed.

Number of lists: 3
List 17
List 18
List 19
List 17 has been removed.

Number of lists: 2
List 18
List 19
List 18 has been removed.

Number of lists: 1
List 19
List 19 has been removed.

Number of lists: 0
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
